platformio.ini

```diff
@@ -130,6 +130,7 @@ lib_deps =
   adafruit/Adafruit PM25 AQI Sensor@^1.0.6
   adafruit/Adafruit MPU6050@^2.2.4
   adafruit/Adafruit LIS3DH@^1.2.4
+  https://github.com/adafruit/Adafruit_AHTX0.git#2.0.5
```

Comment:
if possible use the Platformio registry instead of Github. that would be
`adafruit/Adafruit AHTX0@^2.0.5`